src/clj/game/cards/programs.clj

```diff
@@ -529,7 +529,7 @@
 (defcard "Bankroll"
   {:implementation "Bankroll gains credits automatically."
    :events [{:event :successful-run
-             :req (req (not (= "Jak Sinclair" (get-in run [:source-card :title])))) ;; TODO: dirty hack
+             :req (req (not= "09052" (get-in run [:source-card :code]))) ;; TODO: dirty hack
```

Comment:
If Jak Sinclair is reprinted, this breaks. Don't change this.

Reply:
I get your point, although I doubt it will be reprinted.

Beside, [Museum of History](https://github.com/mtgred/netrunner/blob/60bea881e58acf7bb609910695da395c995ca81e/src/clj/game/cards/assets.clj#L1458-L1464) and [SYNC](https://github.com/mtgred/netrunner/blob/d92144863472eb704442d2fdde3d4f86eea4163e/src/clj/game/cards/identities.clj#L1801) also refer to their own card code.